app/components/recruiter-conversations/conversation/status.js

```diff
@@ -8,11 +8,17 @@ export default Ember.Component.extend({
 
   onInit: function() {
     const id = this.get('conversation.id');
-    return this.get('profiles').getMessages(id)
+    return this.get('profiles').getMessagesForConversation(id)
+      .then((messages) => {
+        const unreadMessages = messages.filter((e) => e.read === false);
```

Comment:
same thing as above.
Using `e` as a variable in the filter is confusing. You should always give descriptive names :)
https://github.com/ryanmcdermott/clean-code-javascript#variables